applications/FluidDynamicsApplication/python_scripts/fluid_dynamics_analysis.py

```diff
@@ -0,0 +1,232 @@
+from __future__ import absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+from KratosMultiphysics import *
```

Comment:
wouldn't it be better to use `import KratosMultiphysics` here?
Since this script won't be modified by "inexpericed users"